mpr-data-access.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "mysql+pymysql://mwrite_peer_review_research_ro:j5L%nKVKayHCdzYL@tl-prod-mysql.aws.vdc.it.umich.edu:3306/mwrite_peer_review\n"
```

Comment:
The commit of this new file has exposed the credentials (password) of the MPR DB to the _public_ repo on GH.  We need to reset the password of this DB ASAP.  I'll open a ticket to do this.

Reply:
I'm really sorry about this. That Jupyter NB was not even used after the first day or so because of the risk of exposing credentials. I need to be more careful about what VSCode stages for syncing.

Reply:
Some git clients try to be extremely helpful with committing as much as possible.  It can be time-consuming, but it's worth it to double check everything before committing or pushing it to the online repo.

Reply:
My advice WRT this kind of thing:

1. Any output in a Jupyter notebook can be saved as part of the notebook.
2. Check the outputs before saving copies of the notebook anywhere.  Remove compromising info.
3. Some Jupyter implementations let you remove some or all output cells.  Some let you specify that the output for specific cells should never be saved.
4. If you print a password for debugging purposes, once you're certain it's working, remove the print statement ASAP.

Reply:
Noted. I'll be more aware of this moving forward.